projects/dex-ui/src/components/Liquidity/LiquidityRoot.tsx

```diff
@@ -0,0 +1,105 @@
+import React, { useState } from "react";
+import styled from "styled-components";
+import { Well } from "@beanstalk/sdk/Wells";
+import { AddLiquidity } from "./AddLiquidity";
+import { RemoveLiquidity } from "./RemoveLiquidity";
+import SlippagePanel from "./SlippagePanel";
+
+type LiquidityRootProps = {
+  well: Well;
+  txnCompleteCallback: () => void;
+};
+
+export const LiquidityRoot = ({ well, txnCompleteCallback }: LiquidityRootProps) => {
+  const [showRemove, setShowRemove] = useState<boolean>(false);
+
+  // Slippage-related
+  const [showSlippageSettings, setShowSlippageSettings] = useState<boolean>(false);
+  const [slippage, setSlippage] = useState<number>(0.1);
+
+  const slippageSettingsClickHandler = () => {
+    setShowSlippageSettings(true);
+  };
+
+  const slippageSettingsCloseHandler = () => {
+    setShowSlippageSettings(false);
+  };
+
+  const handleSlippageValueChange = (value: string) => {
+    console.debug(`Slippage changed: ${parseFloat(value)}`);
```

Comment:
debug

Reply:
I can rebase and use the logger once your other PR is merged.